lisa/transformers/rpm_kernel_installer.py

```diff
@@ -18,10 +19,18 @@
 @dataclass_json()
 @dataclass
 class RPMInstallerSchema(BaseInstallerSchema):
-    # kernel rpm - local absolute path
-    kernel_rpm_path: str = field(
+    # kernel rpms list - local absolute path
+    rpm_list: List[str] = field(
```

Comment:
It needs to have two transformers. This one is for kernel installation, there are extra steps. It needs to add another transformer called `PackageInstallationTransformer`, and derive a transformer called `RpmPackageInstallationTransformer`.

Reply:
Sure, I will add the suggested transformers for installation of non-kernel packages. But even for kernel, there could be multiple kernel related packages one wants to install along with the kernel itself for it to work properly. For example, following could be the list of rpms.
* kernel-some_version.rpm
* kernel-devel-some_version.rpm
* kernel-debuginfo-some_version.rpm
* kernel-docs-some_version.rpm
* kernel-tools-some_version.rpm
etc
Hence, the `List` type to support installing these set of kernel packages together.

Reply:
Only the first one is the kernel RPM, others are not. Other packages can be installed by another transformer, before or after the kernel transformer.

Reply:
Sure. I was of the opinion that kernel related packages should be installed together.
I will then keep the Kernel RPM installer as it is and add the new installers as suggested. Thanks.

Reply:
Hi @squirrelsc . I'm thinking of following schema for the `PackageInstallationTransformer`. Please let me know if this looks good to you.

```
- type: package_installer
  phase: <PHASE> # should also support 'environment_connected'
  connection:
    .
    .
  reboot: <true|false>
  installer:
    type: local # install local packages
    file_list: <LIST>
    package_manager: 
      type: <rpm|apt|..>
      <additional parameters specific to each manager. For example: force_install>

  #  # Other extendable installers could be
  # installer:
  #   type: remote # install remote packages
  #   packages: <LIST>
         
```